UITests/TestBase.cs

```diff
@@ -0,0 +1,50 @@
+﻿//******************************************************************************
+//
+// Copyright (c) 2023 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+using OpenQA.Selenium.Interactions;
+using System.Threading;
+
+namespace UITests
+{
+	public class TestBase
+	{
+		public static WindowsDriver<WindowsElement> Session => SessionManager.Session;
+
+		public static void OpenControlPage(string name)
+		{
+			var search = Session.FindElementByName("Search");
+			search.Clear();
+			Thread.Sleep(1_000);
+			search.SendKeys(name);
+			Thread.Sleep(1_000);
+			Session.FindElementByName(name).Click();
+			Thread.Sleep(5_000);
+			AxeHelper.AssertNoAccessibilityErrors();
```

Comment:
We should add explicit Axe test methods, instead of having this validation piggy-back on other tests via this helper method.

Reply:
E.g. Tests\Button.cs should be updated to have
```
[TestMethod]
public void ValidateAccessibilityWithAxe()
```
or something like that.